ansible/workers.yml

```diff
@@ -156,10 +200,16 @@
       - cmake
       - openscap-utils
       - wget  # needed for the NIST test suite for SSG
-      - PyYAML  # needed for the new SSG yaml port
       - ansible  # needed for ansible role syntax check
       state: installed
 
+  - name: Ensure SCAP Security Guide dependencies are installed
+    package:
+      name:
+      - PyYAML  # needed for the new SSG yaml port
+      state: installed
+    when: ansible_distribution != 'RedHat' or ansible_distribution_version.split(".")[0] != "8"
```

Comment:
Is assume this task RHEL8 only, as next task is Fedora only.
So shouldn't this be `ansible_distribution == 'RedHat' and ansible_distribution_version.split(".")[0] != "8"`?